scan.go

```diff
@@ -42,6 +42,7 @@ var (
 	githubRepoUrls                   []string
 	githubOrgUrl                     string
 	githubOrgMaxRepositories         int
+	githubOrgExcludeRepos            string
```

Comment:
This should be a `[]string` because we are accepting multiple values for this param